base/card-api.gts

```diff
@@ -597,11 +637,13 @@ class DefaultEdit extends GlimmerComponent<{ Args: { model: Card; fields: Record
     </style>
     <div class="default-edit">
       {{#each-in @fields as |key Field|}}
-        <label data-test-field={{key}}>
-          {{!-- @glint-ignore glint is arriving at an incorrect type signature --}}
-          {{startCase key}}
-          <Field />
-        </label>
+        {{#unless (eq key 'id')}}
+          <label data-test-field={{key}}>
+            {{!-- @glint-ignore glint is arriving at an incorrect type signature --}}
+            {{startCase key}}
+            <Field />
+          </label>
+        {{/unless}}
```

Comment:
i just noticed this: the contained cards edit format render with an "id" field. seems yucky. this eliminates the id field for the non contained cards. but I figured we'll work thru a  better approach for that as we move forward with the links to work.